docdoku-server/docdoku-server-web/src/main/java/com/docdoku/server/rest/dto/LightPartRevisionDTO.java

```diff
@@ -0,0 +1,44 @@
+package com.docdoku.server.rest.dto;
```

Comment:
If we use the classic one , the unused properties will be sent with null value. The response will be heavier and will contains many useless data. This class is useful because it is lighter and is easier to manipulate.